### PR TITLE
[TASK 3] added checking on valid data and bike ID

### PR DESCRIPTION
**Pull request by iprisakaru**
1) Added checking time and data on conformity to type "YYYY-MM-DD HH:MM:SS"
2) Added checking bike ID on conformity to type " letter + 6 numbers "

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +0,0 @@
-### First Lab on Scala
-
-
-- **Technology:**
```
Comment (alexander-belyaev) [inline README.md:4]: Why did you decide to remove Readme.md?

File: src/main/scala/by/bsu/FileService.scala [hunk 2]
```diff
@@ -20,12 +21,22 @@ object FileService {
   def readFileCVS(fileName: String): (Int, List[Trip]) = {
     val outputString: List[Either[Exception, Trip]] = fromFile(fileName).getLines().map(_.replaceAll("\"", "")
       .split(",").map(_.trim)).map(line => {
-      if (line.length != 9) Left(new IllegalArgumentException) else Right(Trip(line(0).toInt, line(1), line(2), line(3).toInt,
+      if (line.length != 9 || !isDataValid(line(1)) || !isDataValid(line(2)) || !isBikeIDValid(line(7))) Left(new IllegalArgumentException) else Right(Trip(line(0).toInt, line(1), line(2), line(3).toInt,
```
Comment (alexander-belyaev) [inline src/main/scala/by/bsu/FileService.scala:24]: Could you clarify what does 1, 2, 3 indexes here mean?

File: src/main/scala/by/bsu/FileService.scala [hunk 3]
```diff
@@ -20,12 +21,22 @@ object FileService {
...
         line(4), line(5).toInt, line(6), line(7), line(8), line(1).substring(5, 7).toByte))
     }).toList
 
+    val a = isBikeIDValid("W35343")
```
Comment (alexander-belyaev) [inline src/main/scala/by/bsu/FileService.scala:28]: Wrong variable naming
Please check the following article:
https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29

File: src/main/scala/by/bsu/FileService.scala [hunk 4]
```diff
@@ -20,12 +21,22 @@ object FileService {
...
     (outputString.count(_.isLeft), outputString.flatMap(_.toOption))
 
   }
 
+  def isDataValid(line : String): Boolean={
```
Comment (alexander-belyaev) [inline src/main/scala/by/bsu/FileService.scala:33]: Please fix formatting near braces

File: src/main/scala/by/bsu/FileService.scala [hunk 5]
```diff
@@ -20,12 +21,22 @@ object FileService {
...
+    line.matches("^([0-9]{4})-([0-9]{2})-([0-9]{2}) ([0-9]{2}):([0-9]{2}):([0-9]{2})$")
+  }
+
+  def isBikeIDValid(line : String): Boolean={
```
Comment (alexander-belyaev) [inline src/main/scala/by/bsu/FileService.scala:37]: Formmatting

**Review by alexander-belyaev — CHANGES_REQUESTED** [review]

File: src/main/scala/by/bsu/FileService.scala (hunk 3 above)
Comment (iprisakaru) [reply src/main/scala/by/bsu/FileService.scala:28]: I just checked how my function work. I'll delete this line.

File: src/main/scala/by/bsu/FileService.scala (hunk 2 above)
Comment (iprisakaru) [reply src/main/scala/by/bsu/FileService.scala:24]: I am given a string as input. I split it into an array of strings. And each element of the array is some kind of class field.

**Review by alexander-belyaev — APPROVED** [review]

**Pull request merged**